guides/common/modules/proc_exporting-a-repository.adoc

```diff
@@ -34,14 +34,14 @@ For more information, see xref:Download_Policies_Overview_{context}[].
 * Ensure that you synchronize products that you export to the required date.
 
 .Procedure
-. Export a repository using the product name and repository name:
+. Export a repository using the product, repository, and organization names:
 +
 [options="nowrap" subs="+quotes"]
 ----
 # hammer content-export complete repository \
---name="_My_Repository_" \
---organization="_My_Organization" \
---product="_My_Product_"
+--name="My_Repository" \
+--product="My_Product"  \
+--organization="My_Organization"
```

Comment:
This does not address the problem reported in https://issues.redhat.com/browse/SAT-26583.

Reply:
@asteflova can you clarify?

I was talking with @jeremylenz about this on Slack and he said from what he was reading, the issue was outlined in your comment. He said this PR is comparing it to the master branch and that it only shows I'm changing the order of the organization and product lines, since the `--organization="My_Organization"` is already there.

He says if this PR is cherry-picked into older branches as well (3.8?) it may result in the `--organization="My_Organization"` being added, rather than reordered.

Reply:
That's a good point @bangelic. The --organization flag is missing on branches 3.8 and lower; but adding it to master and cherry-picking all the way down will ensure that the procedure will be consistent across versions, which is a good thing.

I'd like you to change one tiny thing: It seems you have accidentally dropped the italics for replaceables on lines 42-44. Can you re-add them? Then your PR should be good to merge.